.travis.yml

```
@@ -3,11 +3,11 @@
 sudo: false
 language: node_js
 node_js:
+  - '6'
   - '8'
-  - '9'
   - '10'
-  - '11'
   - '12'
+  - 'node'
```

Why this line ? to test on current stable release ? it should be `'stable'` in this case

---

`stable` and `node` are the same for TravisCI afaik

---

```suggestion
  - 'stable'
```
So I suggest to use `'stable'` because it’s more explicite.